content/docs/command-reference/plots/show.md

```diff
@@ -174,10 +174,60 @@ file:///Users/usr/src/plots/logs.csv.json
 
 ```json
 {
-    "$schema": "https://vega.github.io/schema/vega-lite/v4.json",
-    "data": {
-        "values": [
-    {
-        "accuracy": "0.9418667",
-    ...
+  "$schema": "https://vega.github.io/schema/vega-lite/v4.json",
+  "data": {
+    "values": [{ "accuracy": "0.9418667", ... }]
+  }
+}
+```
+
+## Example: Smooth plot
+
+In some cases we would like to smooth our plot. In this example we will use a
+plot with 1000 data points:
+
+```dvc
```

Comment:
Hmmmm. Where does this come from @jeremydesroches? Seems useful but totally out of scope for this PR?

Reply:
Ooooh I see it was moved here from the index. Yeah that's not necessary, we have some mixed examples in cmd indices some times. Let's put all those back please 🙂

Reply:
OK, reverted.

Reply:
Thanks